editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -857,7 +868,10 @@ void EditorAssetLibrary::_image_request_completed(int p_status, int p_code, cons
 		_image_update(p_code == HTTPClient::RESPONSE_NOT_MODIFIED, true, p_data, p_queue_id);
 
 	} else {
-		WARN_PRINT("Error getting image file from URL: " + image_queue[p_queue_id].image_url);
+		if (is_print_verbose_enabled()) {
+			WARN_PRINT(vformat("Asset Library: Error getting image from '%s' for asset # %d.", image_queue[p_queue_id].image_url, image_queue[p_queue_id].asset_id));
+		}
```

Comment:
Makes me think we want to add a dedicated macro for this, a `WARN_PRINT_VERBOSE` or something

Reply:
So far this is very situational, and normally we want all errors and warnings to be always printed. But yes, I was thinking about this as well.

Reply:
We have a `WARN_PRINT_ONCE` so a macro for `WARN_PRINT_VERBOSE` could be adapted to that too, but good to discuss in the future

Reply:
👍

The reason why I mentioned a dedicated local method is because I want `WARN_PRINT_ONCE` to be called once for all warnings from all other methods. Since `WARN_PRINT_ONCE` works based on a scope, I need to create a funnel for all messages.